server/sonar-process-monitor/src/main/java/org/sonar/process/monitor/Monitor.java

```diff
@@ -282,8 +282,12 @@ private RestartorThread() {
 
     @Override
     public void run() {
-      stopProcesses();
-      startProcesses();
+      try {
+        stopProcesses();
```

Comment:
no need to wrap both `stopProcesses()` and `startProcesses()`, only the latter can throw `InterruptedException`
also, a comment in the `catch` would be wise (that we just ignore the interrupt for now)